Tests/TuistGeneratorIntegrationTests/Generator/TestData/GeneratedSchemeDescriptor+TestData.swift

```diff
@@ -0,0 +1,12 @@
+
+import Basic
+import Foundation
+import XcodeProj
+@testable import TuistGenerator
+
+extension SchemeDescriptor {
```

Comment:
Considering `SchemeDescriptor` is a public model, I'd move this to the `TuistGeneratorTesting` framework so that the dependant frameworks can have access to this method too.